src/ext/DifxApp/wixext/WixToolset.DifxApp.wixext.csproj

```diff
@@ -8,15 +8,13 @@
     <Description>WiX Toolset DIFxApp Extension</Description>
     <Title>WiX Toolset DIFxApp Extension</Title>
     <DebugType>embedded</DebugType>
+    <NuspecFile>$(MSBuildThisFileName).nuspec</NuspecFile>
```

Comment:
You don't need to switch to .nuspec any more. Additional content can be included with `Content` items.